Automattic/jetpack
# Add advanced menu items for General and Writing for WP Admin

Fixes https://github.com/Automattic/wp-calypso/issues/49191

#### Changes proposed in this Pull Request:

* When the unified nav is enabled on Atomic sites, add "Advanced General" and "Advanced Writing" menu items under Settings when the user has Calypso links enabled.

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Install Jetpack Beta on an Atomic site and active this branch
* Go to wordpress.com/me/account and ensure that "Dashboard appearance" toggle is ON 
* On WP.com switch to that site and observe the new menu items "Advanced General" and "Advanced Writing" under "Settings" in the menu and link to their WP Admin General and Writing counterparts.
* Go to wordpress.com/me/account and ensure that "Dashboard appearance" toggle is OFF
* on WP.com switch to that site and check that the menu items are now NOT present in the menu.

![Screenshot 2021-02-15 at 13 26 08](https://user-images.githubusercontent.com/8639742/107952789-1ef1bf00-6f92-11eb-9142-67ffae679837.png)

#### Proposed changelog entry for your changes:


* Admin Menu: Add "Advanced General" and "Advanced Writing" sub-menu items on WoA sites
